alpha/apps/kaltura/lib/reports/myReportsMgr.class.php

```diff
@@ -1503,6 +1503,7 @@ class reportsInputFilter
 	public $extra_map;
 	public $categories;
 	public $categoriesIds;
+	public $custom_var1;
```

Comment:
Better use one format of member names, e.g. lowerCamelCase.

Reply:
Yes, I saw that... however, this class already uses mixed conventions (before this pull) so I just went with the majority for now, which is underscores